src/grammar/Fpp.g4

```diff
@@ -180,6 +233,12 @@ componentMemberTempl:
     | matchStmt
     | recordSpecifierDecl
     | containerSpecifierDecl
+    | stateMachineInstance
+
+    // FIXME(tumbar) Should I enable this syntax?
+    // technically this is not allowed here but we will allow this syntax and warn later in the analysis
+    // state machine definitions are only allowed in module definitions
```

Comment:
Actually state machine definitions are allowed as component members. There is an omission in the spec. I opened an issue on this: https://github.com/nasa/fpp/issues/551.